context/app/api/vitessce.py

```diff
@@ -115,26 +116,30 @@
 CODEX_TILE_PATH = "output/extract/expressions/ome-tiff"
 CODDEX_SPRM_PATH = "output_json"
 
+SEQFISH_HYB_CYCLE = r"HybCycle_\d+"
+SEQFISH_NAME = r"MMStack_Pos\d+\.ome\.tiff?"
```

Comment:
Unless there's another organizing principle, define these right before they are used. Consider `_RE`  in name.